### DISCUSSION TITLE
will the example on cocoapods be updated?

### DISCUSSION THREAD

**Discussion by Daymox** [AndyQ/NFCPassportReader#222] [General]
Hi!

Currently, I'm working with the CocoaPods version and I would like to know if maintenance will continue, especially regarding the OpenSSL v3.1.5004

**Comment by AndyQ** [comment]
I'm probably going to deprecate/remove support for Cocoapods soon - esp in the light of it moving to purely maintenance mode. Would be interested to see if this is considered a big issue though.
